src/routes/MovieListPage/MovieListPage.tsx

```diff
@@ -26,29 +25,30 @@ export async function loader({ request }) {
     limit: 9,
   };
 
-  const movies = await getMovies(params, { signal: controller.signal }).then(({ data }) => {
-    const movies = data.map((movie) => convertToMovieModel(movie));
-    controller = null;
-    return movies;
-  }).catch((e) => {
-    if (e.name === 'AbortError') {
-      return;
-    }
-    throw e;
-  })
+  const movies = await getMovies(params, { signal: controller.signal })
+    .then(({ data }: Response<Movie[]>) => {
+      const movies = data.map((movie: Movie) => convertToMovieModel(movie));
+      controller = null;
+      return movies;
+    }).catch((e: Error) => {
+      if (e.name === 'AbortError') {
+        return;
+      }
+      throw e;
+    })
 
   return movies;
 };
 
 export function MovieListPage() {
   const navigate = useNavigate();
   const [searchParams, setSearchParams] = useSearchParams();
-  const movieList = useLoaderData();
+  const movieList: unknown = useLoaderData();
```

Comment:
Not sure that it's right type for `movieList`. Is it possible to create/use `Movie[]` Type/Interface here instead of `unknown`?